tests/testthat/test-multitables.R

```diff
@@ -87,6 +87,29 @@ with_mock_HTTP({
             expect_is(mtable, "Multitable")
         })
     })
+    
+    test_that("importMultitable", {
+        with_POST("https://app.crunch.io/api/datasets/1/multitables/4de322/", {
+            mtable <- newMultitable(~ gender + mymrset, data=ds, name="New multitable")
+            expect_is(mtable, "Multitable")
+        })
+        expect_POST(m <- importMultitable(ds, mtable, name='copied_multitable'),
```

Comment:
For future reference, you don't need to assign (`m <-`) here. The POST request will stop before the function returns.